src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -12,8 +13,84 @@ import computeStateName from "../../../common/entity/compute_state_name";
 export default dedupingMixin(
   (superClass) =>
     class extends NavigateMixin(EventsMixin(superClass)) {
-      handleClick(hass, config) {
-        const tapAction = config.tap_action || "more-info";
+      registerMouse(config) {
+        let ripple = null;
+        const rippleWrapper = document.createElement("div");
+        this.shadowRoot.appendChild(rippleWrapper);
+        rippleWrapper.style.position = "absolute";
+        rippleWrapper.style.width = "100px";
+        rippleWrapper.style.height = "100px";
```

Comment:
This might be a bit too much on a computer with a mouse...
Would it make sense to have bigger ripples on a touch screen, or do we want to keep the interface absolutely consistent?

Reply:
After trying it out; screw consistency. It's grotesquely big on a computer. I'll fix this.